library/SKETCHOPS

```diff
@@ -600,29 +598,31 @@ Copyright (c) 1986, 1987, 1990, 1992, 1993 by Venue & Xerox Corporation.  All ri
 (DEFINEQ
 
 (DRAW.LOCAL.SKETCH
-  [LAMBDA (LOCALSPECS STREAM STREAMREGION SCALE)         (* ; "Edited 24-Mar-92 14:00 by jds")
+  [LAMBDA (LOCALSPECS STREAM STREAMREGION SCALE)             (* ; "Edited  2-May-2023 13:28 by lmm")
+                                                             (* ; "Edited 24-Mar-92 14:00 by jds")
 
     (* ;; "draws the local specs on a stream")
 
     (* ;; "set priority of the stream in case mode is set to REPLACE or ERASE --- would be better to scan list looking for an element that actually has one of these.")
 
+    (MOVD? 'NILL 'SET.PRIORITYIMPORTANT)
```

Comment:
Why does this make sense when SET.PRIORITYIMPORTANT is defined in this file?  If there's a reason for it, how about a comment explaining?

Reply:
The MOVD? was unnecessary. I mistakenly thought the function was on INTERPRESS
Now gone.